tools/fetch_fasta_from_ncbi/retrieve_fasta_from_NCBI.py

```diff
@@ -310,12 +329,27 @@ def __main__():
     parser.add_option('-i', dest='query_string', help='NCBI Query String')
     parser.add_option('-o', dest='outname', help='output file name')
     parser.add_option('-l', '--logfile', help='log file (default=stderr)')
+    parser.add_option('--datetype', dest='datetype', help='Type of date used to limit a search. [modification_date, publication_date, entrez_date] (default=entrez_date)', default='entrez_date')
+    parser.add_option('--reldate', dest='reldate', help='When reldate is set to an integer n, the search returns only those items that have a date specified by datetype within the last n days.')
+    parser.add_option('--maxdate', dest='maxdate', help='Date range used to limit a search result by the date specified by datetype. These two parameters (mindate, maxdate) must be used together to specify an arbitrary date range. The general date format is YYYY/MM/DD, and these variants are also allowed: YYYY, YYYY/MM.')
+    parser.add_option('--mindate', dest='mindate', help='Date range used to limit a search result by the date specified by datetype. These two parameters (mindate, maxdate) must be used together to specify an arbitrary date range. The general date format is YYYY/MM/DD, and these variants are also allowed: YYYY, YYYY/MM.')
     parser.add_option('--loglevel', choices=LOG_LEVELS, default='INFO', help='logging level (default: INFO)')
     parser.add_option('-d', dest='dbname', help='database type')
     (options, args) = parser.parse_args()
     if len(args) > 0:
         parser.error('Wrong number of arguments')
-    
+    if (options.reldate and options.maxdate) or (options.reldate and options.mindate):
```

Comment:
Can you enforce users not mixing parameters by manipulating the arguments ?

On a side note, the xml plays the role of keeping parameters consistant. So it may be unnecessary to deal with this